optaplanner-benchmark/src/build/revapi-config.json

```diff
@@ -0,0 +1,118 @@
+        "justification": "ADD YOUR EXPLANATION FOR THE NECESSITY OF THIS CHANGE"
+      },
+      {
+        "code": "java.method.numberOfParametersChanged",
+        "old": "method void org.optaplanner.benchmark.config.ProblemBenchmarksConfig::buildProblemBenchmarkList(org.optaplanner.benchmark.impl.result.SolverBenchmarkResult)",
+        "new": "method void org.optaplanner.benchmark.config.ProblemBenchmarksConfig::buildProblemBenchmarkList(org.optaplanner.core.config.SolverConfigContext, org.optaplanner.benchmark.impl.result.SolverBenchmarkResult)",
+        "justification": "ADD YOUR EXPLANATION FOR THE NECESSITY OF THIS CHANGE"
+      },
+      {
+        "code": "java.method.numberOfParametersChanged",
+        "old": "method void org.optaplanner.benchmark.config.SolverBenchmarkConfig::buildSolverBenchmark(org.optaplanner.benchmark.impl.result.PlannerBenchmarkResult)",
+        "new": "method void org.optaplanner.benchmark.config.SolverBenchmarkConfig<Solution_>::buildSolverBenchmark(org.optaplanner.core.config.SolverConfigContext, org.optaplanner.benchmark.impl.result.PlannerBenchmarkResult)",
+        "justification": "ADD YOUR EXPLANATION FOR THE NECESSITY OF THIS CHANGE"
+      },
+      {
+        "code": "java.generics.formalTypeParameterAdded",
+        "old": "class org.optaplanner.benchmark.config.SolverBenchmarkConfig",
+        "new": "class org.optaplanner.benchmark.config.SolverBenchmarkConfig<Solution_ extends java.lang.Object>",
+        "justification": "ADD YOUR EXPLANATION FOR THE NECESSITY OF THIS CHANGE"
+      },
+      {
+        "code": "java.field.removed",
+        "old": "field org.optaplanner.benchmark.config.blueprint.SolverBenchmarkBluePrintType.ALL_CONSTRUCTION_HEURISTIC_TYPES",
+        "justification": "ADD YOUR EXPLANATION FOR THE NECESSITY OF THIS CHANGE"
+      },
+      {
+        "code": "java.missing.newClass",
+        "new": "missing-class com.google.errorprone.annotations.CanIgnoreReturnValue",
+        "justification": "ADD YOUR EXPLANATION FOR THE NECESSITY OF THIS CHANGE"
+      },
+      {
+        "code": "java.missing.oldClass",
+        "old": "missing-class sun.java2d.pipe.hw.ExtendedBufferCapabilities",
+        "new": "missing-class sun.java2d.pipe.hw.ExtendedBufferCapabilities",
+        "justification": "Probably a bug in revapi, nothing serious"
+      },
+      {
+        "code": "java.missing.newClass",
+        "old": "missing-class sun.java2d.pipe.hw.ExtendedBufferCapabilities",
+        "new": "missing-class sun.java2d.pipe.hw.ExtendedBufferCapabilities",
+        "justification": "Probably a bug in revapi, nothing serious"
+      },
+      {
+        "code": "java.missing.oldClass",
+        "old": "missing-class sun.util.locale.provider.LocaleServiceProviderPool.LocalizedObjectGetter",
+        "new": "missing-class sun.util.locale.provider.LocaleServiceProviderPool.LocalizedObjectGetter",
+        "justification": "Probably a bug in revapi, nothing serious"
```

Comment:
Can we filter out these bogus messages (anything not in org.optaplanner basically) in the plugin configuration?